pkg/reconciler/buildrun/resources/failure_details.go

```diff
@@ -62,8 +63,8 @@ func extractFailureReasonAndMessage(taskRun *v1beta1.TaskRun) (errorReason strin
 	return errorReason, errorMessage
 }
 
-func extractFailedPodAndContainer(ctx context.Context, client client.Client, taskRun *v1beta1.TaskRun) (*v1.Pod, *v1.Container, error) {
-	var pod v1.Pod
+func extractFailedPodAndContainer(ctx context.Context, client client.Client, taskRun *v1beta1.TaskRun) (*corev1.Pod, *corev1.Container, error) {
+	var pod corev1.Pod
 	if err := client.Get(ctx, types.NamespacedName{Namespace: taskRun.Namespace, Name: taskRun.Status.PodName}, &pod); err != nil {
 		return nil, nil, err
```

Comment:
Should we log directly here ?

Reply:
Well, I do not have a strong opinion here. I can change it. I figured this way we can get rid of the `pod, container, _ := ...` statement that explicitly omits the error, which looks not so nice.